content/kotlin/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,390 @@
+---
+Title: 'Data-Types'
+Description: 'In Kotlin, a data type is a classification of types of values that can be stored and manipulated in a program.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Primitive Data-Type'
+  - 'Non Primitive Data-Type'
+  
+CatalogContent:
+  - 'learn-Kotlin'
+  - 'paths/computer-science'
+---
+
+A data type is a classification of types of data that determine the possible values and operations that can be performed on that data.
+
+## Type of Data-Types 
+
+Kotlin has both primitive and non-primitive data type.
+1. Primitive Data types
+2. Non-primitive Data types 
+
+## Primitive Data-type
+
+We will now discuss different types of primitive data-types in kotlin :
+
+- **Boolean**
+- **Byte**
+- **Char**
+- **Double**
+- **Float**
+- **Int**
+- **Long**
+- **Short**
+
+
+# Boolean 
```

Comment:
Two things that apply to all the following:

1. There should only be one line break before and after headers and code blocks.
2. The sub heads should be preceded by three hash marks, by using only one, you're styling these as &lt;h1&gt;